code/modules/antagonists/ashwalker/ashwalker.dm

```diff
@@ -42,3 +38,40 @@
 
 	if(istype(A, /obj/structure/headpike))
 		owner.current.add_mood_event("oogabooga", /datum/mood_event/sacrifice_good)
+
+/datum/team/ashwalkers
+	name = "Ash Walker Tribe"
+	member_name = "Ash Walker"
+	///A list of "worthy" (meat-bearing) sacrifices made to the Necropolis
+	var/sacrifices_made = 0
+	///A list of how many eggs were created by the Necropolis
+	var/eggs_created = 0
+
+/datum/team/ashwalkers/roundend_report()
+	var/list/report = list()
+
+	report += "<span class='header'>An Ash Walker Tribe inhabited the wastes...</span><br>"
```

Comment:
do you think it'd be worthwhile to make a `span_header()` for this in this PR? it should definitely be done at some point so i'm wondering if now is a good time or not.

Reply:
im killing this whole menu when i get the motivation anyway